packages/react/src/components/table/table.tsx

```diff
@@ -168,31 +171,62 @@ const ExpandButton = styled(IconButton) <{ $expanded: boolean }>`
     }
 `;
 
-function getUtilityColumn<T extends object>(type: UtilityColumnType, t: TFunction<'translation'>): TableColumn<T> {
+function getUtilityColumn<T extends object>(
+    type: UtilityColumnType,
+    t: TFunction<'translation'>,
+    rowSelectionMode?: RowSelectionMode,
+    ariaLabelledByColumnId?: string,
+): TableColumn<T> {
     const column: TableColumn<T> = {
         id: type,
         className: utilColumnClassName,
     };
 
     switch (type) {
         case 'selection':
-            column.header = ({ table }) => (
-                <Checkbox
-                    data-testid="row-checkbox-all"
-                    checked={table.getIsAllRowsSelected()}
-                    indeterminate={table.getIsSomeRowsSelected()}
-                    onChange={table.getToggleAllRowsSelectedHandler()}
-                />
-            );
-            column.cell = ({ row }) => (
-                <Checkbox
-                    data-testid={`row-checkbox-${row.index}`}
-                    checked={row.getIsSelected()}
-                    disabled={!row.getCanSelect()}
-                    indeterminate={row.getIsSomeSelected()}
-                    onChange={row.getToggleSelectedHandler()}
-                />
-            );
+            if (rowSelectionMode === 'multiple') {
+                column.header = ({ table }) => (
+                    <Checkbox
+                        data-testid="row-checkbox-all"
+                        checked={table.getIsAllRowsSelected()}
+                        indeterminate={table.getIsSomeRowsSelected()}
+                        onChange={table.getToggleAllRowsSelectedHandler()}
+                    />
+                );
+                column.cell = ({ row }) => (
+                    <Checkbox
+                        data-testid={`row-checkbox-${row.index}`}
+                        checked={row.getIsSelected()}
+                        disabled={!row.getCanSelect()}
+                        indeterminate={row.getIsSomeSelected()}
+                        onChange={row.getToggleSelectedHandler()}
+                    />
+                );
+            } else if (rowSelectionMode === 'single') {
+                if (!ariaLabelledByColumnId) {
+                    console.warn('ariaLabelledByColumnId is recommended for Accessibility');
+                }
+
+                column.cell = ({ table, row }) => {
+                    const radioBtnId = `row-radiobutton-${row.index}`;
+                    return (
+                        <RadioButton
```

Comment:
En lien avec le commentaire plus haut concernant le `margin-top` dans le `radio-button.tsx`
```suggestion
const StyledRadioButton = styled(RadioButton)`
    margin-top: 0;
`
...
<StyledRadioButton...
```